TWLight/users/views.py

```diff
@@ -632,8 +636,8 @@ def get_success_url(self):
             # put them in an obnoxious redirect loop - send them to where
             # they were going, which promptly sends them back to the terms
             # page because they haven't agreed to the terms....
-            # Also clear the ToU date field in case a user un-agrees
-            self.get_object().terms_of_use_date = None
+            # However, we will keep the terms_of_use_date as last updated date
+            self.get_object().terms_of_use_date = datetime.date.today()
```

Comment:
Should this date be localized for the user? As written, it will be app server time, right?

Reply:
Dates/times are stored in UTC per https://github.com/WikipediaLibrary/TWLight/blob/fcc78682fc4fa14009007d06471138a6edef328e/TWLight/settings/base.py#L270

This date isn't shown to users anywhere, so we don't need to worry about confusing them. Provided we understand internally that the date is UTC, I think this is fine.

Reply:
Excellent. Thanks for clearing that up.

Reply:
Thank you for clarifying.  Previously, I was working on data migration for updating terms_of_use_date to the DateTime field.  So we can display the date in the client's timezone.  Maybe we can revisit date related tasks in the future.

I reverted the code.  The date won't be displayed but will be captured.  It can be viewed on the admin page.  Please let me know if you have any questions.

![](https://i.imgur.com/KikNLXE.png)